src/css/App.css

```diff
@@ -558,21 +562,14 @@ footer {
 }
```

Comment:
To be a pain, would you mind editing the class above to not have the `overflow-y: hidden` - from what I can tell, this causes a horizontal scroll bar, and is never needed.

Reply:
I went even a step further and made the nav menu responsive down to about 200px by turning the FCC logo into just the icon at very narrow widths.